README.md

```diff
@@ -66,4 +57,4 @@ Digital Bazaar: support@digitalbazaar.com
 
 ## License
 
-[New BSD License (3-clause)](LICENSE) © Digital Bazaar
+[Bedrock Non-Commercial License v1.0](LICENSE.md) © Digital Bazaar
```

Comment:
Which license is this supposed to have?  The BSD "LICENSE" exists now, not the Bedrock "LICENSE.md".

Reply:
I am not sure about this. I believe this was copied from bedrock-web-vc-store. If it needs to be changed, let me know.

Reply:
Since this is a bedrock* package, I'm guess the proper fix is to replace BSD LICENSE file with the Bedrock LICENSE.md file.